src/scene/container/container-mixins/effectsMixin.ts

```diff
@@ -48,6 +54,15 @@ export const effectsMixin: Partial<Container> = {
      */
     effects: [],
 
+    _markStructureAsChanged()
```

Comment:
Could you add private to this?

```suggestion
    private _markStructureAsChanged()
```